dashboard/test/models/user_test.rb

```diff
@@ -192,6 +192,218 @@ class UserTest < ActiveSupport::TestCase
     assert_equal ['Email has already been taken'], user.errors.full_messages
   end
 
+  #
+  # Email uniqueness validation tests
+  # These should simplify significantly once we are fully migrated to multi-auth
+  #
+  COLLISION_EMAIL = 'collision@example.org'
+
+  def create_single_auth_user_with_email(email)
```

Comment:
should all of these helper methods be private? i've seen it done both ways, so i'm not sure if we have a convention/preference

Reply:
I don't think we particularly care in tests. :man_shrugging: Here, I think there's more value in roughly colocating the helpers with the tests that use them, than in grouping the private methods together.